### DISCUSSION TITLE
Is there still any hope to block video after the recent youtube change?

### DISCUSSION THREAD

**Discussion by rasputozen** [Thunderarea/music-mode-for-youtube#41] [Q&A] [answered]
I could save so much bandwidth if there was a way to block video streams. Is all hope lost? Has the pursuit been given up?

**Comment by Thunderarea** [comment, marked as answer]
Currently, there doesn’t seem to be a solution. The video and audio are in the same file and everything that I have tried to get only the audio hasn’t worked. I’ll keep experimenting, but to be honest, I’m not very optimistic.

**Reply by rasputozen** [reply]
Man that's such a bummer. Projects like this help youtube reduce its resource expenditure, and therefore its carbon footprint, and yet they design their streaming in a way that prevents it. They could still do audio-only advertisements. It's just so dumb and wasteful, both for them and for our wallets having to buy more data than we want or need. Thanks for trying my friend.

**Comment by PopGBop** [comment]
It's been a year. Any updates on this inability to separate the audio out from the video stream? Thanks.

**Reply by Thunderarea** [reply]
Unfortunately, there doesn’t seem to be a reliable way to separate the audio from the video. The extension will still work, but it will only hide the video on YouTube rather than block it.
